src/sif/autocost.cc

```diff
@@ -66,11 +69,15 @@ constexpr float kLeftSideTurnCosts[] = {kTCStraight,         kTCSlight,  kTCUnfa
                                         kTCUnfavorableSharp, kTCReverse, kTCFavorableSharp,
                                         kTCFavorable,        kTCSlight};
 
+constexpr float kMinFactor = 0.1f;
+constexpr float kMaxFactor = 100000.0f;
```

Comment:
just wondering how what happens when you actually do a route with min or max, any oddities in perf or path selection?

Reply:
@kevinkreiser as @dgearhart noted.  I followed what pedestrian has set.

Reply:
Opened #2234 so that we check all the costing.